src/test/groovy/io/codearte/jfairy/producer/person/locale/en/EnAddressSpec.groovy

```diff
@@ -9,8 +9,8 @@ import static org.apache.commons.lang3.SystemUtils.LINE_SEPARATOR
 class EnAddressSpec extends Specification {
 
 	private final int SEED = 7
```

Comment:
I missed it yestarday - there `static` could be added to make it "real" constant.